misc/scripts/install-local.sh

```diff
@@ -128,17 +128,16 @@ function log() {
 		echo "_ppa=\"$ppa"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
 	fi
 	if test -f /tmp/pacstall-pacdeps-"$PACKAGE"; then
-		echo "_pacstall_depends=\"true"\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
+		echo '_pacstall_depends="true'\" | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
```

Comment:
```suggestion
		echo '_pacstall_depends="true"' | sudo tee -a "$LOGDIR"/"$PACKAGE" > /dev/null
```
This is the way we should have do all of this metadata writings

Reply:
Oh god, so many